.github/workflows/rust.yml

```diff
@@ -12,10 +12,16 @@ env:
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
 
     steps:
     - uses: actions/checkout@v2
+    - name: Install Rust nightly
+      uses: actions-rs/toolchain@v1
+      with:
+          toolchain: nightly
```

Comment:
Can we pin to today's Nightly date instead of the latest Nightly? Obviously this pin is terrible, but it's better than having to fight with Rust upgrades every time we build it.

`toolchain: nightly-YYYY-MM-DD` should work, based on https://github.com/actions-rs/toolchain